cookbooks/bcpc-hadoop/recipes/hbase_config.rb

```diff
@@ -58,15 +58,23 @@
 #
 generated_values = {
   'hbase.zookeeper.quorum' =>
-    node[:bcpc][:hadoop][:zookeeper][:servers].map{ |s| float_host(s[:hostname])}.join(","),
-  'hbase.zookeeper.property.clientPort' => "#{node[:bcpc][:hadoop][:zookeeper][:port]}",
+    node[:bcpc][:hadoop][:zookeeper][:servers].map do |s|
+      float_host(s[:hostname])
+    end.join(","),
+  'hbase.zookeeper.property.clientPort' =>
+    "#{node[:bcpc][:hadoop][:zookeeper][:port]}",
   'hbase.master.hostname' => float_host(node[:fqdn]),
   'hbase.regionserver.hostname' => float_host(node[:fqdn]),
   'hbase.regionserver.dns.interface' =>
-      node["bcpc"]["networks"][subnet]["floating"]["interface"],
+    node["bcpc"]["networks"][subnet]["floating"]["interface"],
   'hbase.master.dns.interface' =>
-      node["bcpc"]["networks"][subnet]["floating"]["interface"],
-  'dfs.client.read.shortcircuit' => node["bcpc"]["hadoop"]["hbase"]["shortcircuit"]["read"].to_s
+    node["bcpc"]["networks"][subnet]["floating"]["interface"],
+  'hbase.regionserver.ipc.address' =>
```

Comment:
We should find out why these are marked as new -- wasn't this material merged into master in the past?